app/controllers/dashboard_controller.rb

```diff
@@ -2,7 +2,9 @@ class DashboardController < ApplicationController
   before_action :authenticate_user!
 
   def show
-    @user = current_user
+
+    @projects = Project.all.where(user_id: current_user.id)
```

Comment:
```suggestion
    @projects = current_user.projects
```
I think we can express this more concisely through association.